### PR TITLE
Feature path tracer

### PR DESCRIPTION
**Pull request by VZout**

### CODE REVIEW

File: demo/demo_frame_graphs.hpp [hunk 1]
```diff
@@ -88,6 +93,45 @@ namespace fg_manager
 			fg->Setup(rs);
 		}
 
+		// Hybrid raytracing
```
Comment (LaisoEmilio) [inline demo/demo_frame_graphs.hpp]: wrong comment

File: resources/shaders/rt_hybrid.hlsl [hunk 2]
```diff
@@ -277,13 +278,16 @@ void ReflectionHit(inout ReflectionHitInfo payload, in MyAttributes attr)
...
+	const float2 sampled_brdf = brdf_lut.SampleLevel(s0, float2(max(dot(fN, V), 0.01f), roughness), 0).rg;
+
 	//Lighting
 	float3 lighting = shade_pixel(hit_pos, V, albedo, metal, roughness, fN, payload.seed, payload.depth);
 
 	//Reflection in reflections
 	float3 reflection = DoReflection(hit_pos, V, fN, payload.seed, payload.depth + 1);
 
 	float3 specular = reflection * F;
+	//float3 specular = reflection * (kS * sampled_brdf.x + sampled_brdf.y);
```
Comment (LaisoEmilio) [inline resources/shaders/rt_hybrid.hlsl]: We need to remember to uncomment this, therefore I added a comment here 👌

Comment (VZout) [reply resources/shaders/rt_hybrid.hlsl]: donzo

Comment (VZout) [reply resources/shaders/rt_hybrid.hlsl]: donzo

File: demo/demo_frame_graphs.hpp (hunk 1 above)
Comment (VZout) [reply demo/demo_frame_graphs.hpp]: donzo

**Pull request merged**